docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -97,22 +75,32 @@ end group
     group validation & commit
         ECC1 -> ECC1 : validate block signature
         loop for each tx in block
-            ECC1 -> ECC1 : validate tx (client) signature
-
-            ref over ECC1
-                **Validate endorsement policy**
-                TODO: more details
-                Using EnclaveMSP will transparently verify
-                the following steps:
-                    - is endorsing enclave part of Org_i?
-                    - has enclave a proper attestation?
-                    - is this enclave also registered to
-                      perform execution of this chaincode?
-            end ref
-
-            ECC1 -> ECC1 : read-write check
-
-            ECC1 -> ECC1 : mark tx as valid/invalid
+
+            note over ECC1
+                For every FPC transaction, we first perform
+                default validation and then FPC validation
+            end note
+
+            group default validation
+                ECC1 -> ECC1 : perform default validation
+                ECC1 -> ECC1 : validate tx (client) signature
+                ECC1 -> ECC1 : MVCC check
+                ECC1 -> ECC1 : validate endorsements
+            end
+
+            group fpc-vscc
+                ECC1 -> ECC1 : enclave_id <- extract from transaction
+                create participant ERCC order 30
+                ECC1 -> ERCC++ : get_attestation(enclave_id)
+                return attestation
+                ECC1 -> ECC1 : check attestation matches enclave identity
+                ECC1 -> ECC1 : verify enclave signature
```

Comment:
Worth mentioning what the signature is over? (i.e., symmetric to the generation in fpc-cc-invocation; obviously depends also on how you handled my comment above on parsing)